index.html

```diff
@@ -3944,283 +3945,242 @@ <h4>updated</h4>
+            </p>
+            <p>
+A <a>DID Method</a> can define different forms of a <a>DID</a> that are
+logically equivalent. An example is when a <a>DID</a> takes one form prior to
+registration in a <a>verifiable data registry</a> and another form after such
+registration. In this case, the <a>DID Method</a> specification might need to
+express one or more <a>DIDs</a> that are logically equivalent to the resolved
+<a>DID</a> as a property of the <a>DID document</a>. This is the purpose of the
+<code><a>equivalentId</a></code>  property.
+            </p>
+            <p>
 The value of <code>equivalentId</code> MUST be an <a
-data-cite="INFRA#ordered-set">ordered set</a> where each item in the list is a <a
-data-cite="INFRA#string">string</a> that conforms to the rules in Section <a
-href="#did-syntax"></a>.
-                  </dd>
-                  <dd>
-The relationship is a statement that each <code><a>equivalentId</a></code> value
-is logically equivalent to the <code>id</code> property value and thus
-identifies the same <a>DID subject</a>.
-                  </dd>
-                  <dd>
+data-cite="INFRA#ordered-set">ordered set</a> where each item in the list is a
+<a data-cite="INFRA#string">string</a> that conforms to the rules in Section <a
+href="#did-syntax"></a>. The relationship is a statement that each
+<code><a>equivalentId</a></code> value is logically equivalent to the
+<code>id</code> property value and thus identifies the same <a>DID subject</a>.
 Each <code><a>equivalentId</a></code> DID value MUST be produced by, and a form
 of, the same <a>DID Method</a> as the <code>id</code> property value. (e.g.
-<code>did:example:abc</code> == <code>did:example:ABC</code>)
-                  </dd>
-                  <dd>
-A conforming <a>DID Method</a> specification MUST guarantee that each
+<code>did:example:abc</code> == <code>did:example:ABC</code>) A conforming
+<a>DID Method</a> specification MUST guarantee that each
 <code><a>equivalentId</a></code> value is logically equivalent to the
 <code>id</code> property value.
-                  </dd>
-                  <dd>
+            </p>
+            <p>
 A requesting party is expected to retain the values from the <code>id</code> and
 <code><a>equivalentId</a></code> properties to ensure any subsequent
 interactions with any of the values they contain are correctly handled as
 logically equivalent (e.g. retain all variants in a database so an interaction
-with any one maps to the same underlying account). <span class="issue">The
-testability of requesting parties is currently under debate and normative
-statements related to requesting parties may be downgraded in the future from a
-MUST to a SHOULD/MAY or similar language.</span>
-                  </dd>
-                </dl>
-
-                <div class="note" title="Equivalence and equivalentId">
-                  <p>
+with any one maps to the same underlying account).
+            </p>
+            <p>
```

Comment:
```suggestion
            <p class="note" title="Stronger equivalence">
```

Suggest making this paragraph a note, and moving it to the end (to be after the next paragraph).